plugins/html-editor/src/plugin.tsx

```diff
@@ -0,0 +1,36 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { Builder } from '@builder.io/sdk';
+import JoditEditor from 'jodit-react';
+
+
+const config = {
+  useSplitMode: true,
+  enableDragAndDropFileToEditor: true,
+  uploader: {
+    insertImageAsBase64URI: true
+  }
+}
+
+interface TextProps {
+  value: string;
+  onChange: (value: string) => void;
+}
+
+function RichTextEditor(props: TextProps) {
+  return (
+    <JoditEditor
+      value={props.value}
+      onChange={props.onChange}
+      config={config}
+    />
+  );
+}
+
+Builder.registerEditor({
+  /**
+   * Here we override the built-in richtext editor.
+   */
+  name: 'html-editor',
```

Comment:
this should be the name of a type, if you want to override built-in rich text editor you'd want to chose `html` or `richText`